app/templates/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}

```

Comment:
Put a space after the "extends". Helps in readability in the same reason you put two lines between your imports and your class definitions in a python file.